cogs/roles.py

```diff
@@ -249,6 +251,24 @@ def get_target(self, target, guild) -> Tuple[List[disnake.Role], List[disnake.ab
 
         return [role], [channel]
 
+    @commands.check(utils.is_bot_admin_or_mod)
+    @commands.slash_command(name="do_da_thing", description='hodi prdeli', guild_ids=[config.guild_id])
+    async def do_da_thing(self, inter: disnake.ApplicationCommandInteraction):
+        guild = self.bot.get_guild(config.guild_id)
+        logChan = self.bot.get_channel(config.log_channel)
+        for channel in guild.channels:
+            if channel.type == disnake.ChannelType.text:
+                boolik = '-' in channel.name
+                name = channel.name.split('-')[0] if boolik else channel.name
```

Comment:
```suggestion
                name = channel.name.split('-')[0]
```
AFAIK there is nothing wrong with calling split on a string that doesn't contain the separator, `"foo".split('-') == ["foo"]`